File: server/Card.py
class Card:

    def __init__(self, card_id, image, function, description, player_id=None):
        self.id = card_id
        self.image = image
        self.function = function
        self.description = description
        self.player_id = player_id

    def to_dict(self):
        return {
            "id": self.id,
            "image": self.image,
            "function": self.function,
            "description": self.description,
            "player_id": self.player_id
        }

    @classmethod
    def from_dict(cls, data):
        return cls(data["id"], data["image"], data["function"], data["description"], data["player_id"])

    def __repr__(self):
        return (f"Card(id={self.id}, image={self.image}, function={self.function}, "
                f"description={self.description}, player_id={self.player_id})")

    def __eq__(self, other):
        return self.id == other.id

    def __hash__(self):
        return hash(self.id)

File: Risiko/Assets/Scripts/Card.cs
public class Card
{
    
    public string id { get; set; }
    public string image { get; set; }
    public string function { get; set; }
    public string description { get; set; }
    public string player_id { get; set; }

    public Card(string card_id, string image, string function, string description, string playerId)
    {
        id = card_id;
        this.image = image;
        this.function = function;
        this.description = description;
        player_id = playerId;
    }

    public override string ToString()
    {
        return $"Card(id={id}, image={image}, function={function}, description={description}, player_id={player_id})";
    }
}